Wireframe/index.html

```diff
@@ -4,10 +4,59 @@
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+        <header>
+        <h1>Introduction to Git</h1>
+        
+          
+          <article class="article1">
+            <img src="images/git.github.jpg " width="300" height="200"  alt="github" />
+            <h2>
+              What is git?
+            </h2>
+           <p> Git is a DevOps tool used for source code management. Git is an essential tool for software developers and is used in open-source and closed-source projects alike. It helps teams work collaboratively, keep track of changes, and maintain the integrity of their codebase. Learning how to use Git effectively is an important skill for anyone involved in software development. Git helps developers manage and track changes to their codebase over time. It allows you to record and store each change made to a project, making it possible to review, revert, or collaborate on code changes.<br><br>
+          
+          <a href=" https://www.atlassian.com/git/tutorials/what-is-git"> <button class="button-link">Learn More </button></a>
+        </article>
+        </header>
+
+          <main>
+          <section class="container"> 
+          <article>
+           <img src="images/git.png " width="300" height="200"  alt="git" />
+      <h2>
+      Why do developers need Git?
+      </h2>
+    <p>
+      One of the biggest advantages of Git is its branching capabilities. Unlike centralized version control systems, Git branches are cheap and easy to merge. This facilitates the feature branch workflow popular with many Git users. Feature branches provide an isolated environment for every change to your codebase.
+    </p>
+    <a href="https://www.atlassian.com/git/tutorials/why-git#:~:text=One%20of%20the%20biggest%20advantages,every%20change%20to%20your%20codebase."> <button class="button-link">Learn More </button></a>
+  </article>
+
+  <article>
+    <img src="images/branch.png " width="300" height="200"  alt="branch" />
+    <h2>
+      What is a branch in Git?
+    </h2>
+
+    <p>
+      A branch in Git is simply a lightweight movable pointer to one of these commits. The default branch name in Git is master . As you start making commits, you're given a master branch that points to the last commit you made.
+    </p>
+    <a href="https://www.atlassian.com/git/tutorials/using-branches#:~:text=A%20branch%20represents%20an%20independent,staging%20area%2C%20and%20project%20history."> <button class="button-link">Learn More </button></a>
+  </article>
+</section>
+</main>   
+     
+<div class="footer">
```

Comment:
why have you included this div? Could you not have achieved the exact same output by assigning the footer child element the footer class?

Reply:
Hi James. Thank you for your comment. You're right. I excluded the footer div.